roles/openshift_facts/library/openshift_facts.py

```diff
@@ -1102,9 +1104,9 @@ def __init__(self, role, filename, local_facts, additive_facts_to_overwrite=Fals
             )
         self.role = role
         self.system_facts = ansible_facts(module)
-        self.facts = self.generate_facts(local_facts, additive_facts_to_overwrite)
+        self.facts = self.generate_facts(local_facts, additive_facts_to_overwrite, openshift_env)
```

Comment:
Instead of passing openshift_env all around, should we just do a self.openshift_env?